adapter/mysql/mysql.go

```diff
@@ -56,6 +56,28 @@ func (d *MysqlDatabase) DumpTableDDL(table string) (string, error) {
 	return ddl, nil
 }
 
+func (d *MysqlDatabase) Views() ([]string, error) {
+	rows, err := d.db.Query("show full tables where TABLE_TYPE = 'VIEW'")
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	var ddls []string
+	for rows.Next() {
+		var viewName, viewType, definition string
+		if err = rows.Scan(&viewName, &viewType); err != nil {
+			return nil, err
+		}
+		query := fmt.Sprintf("select VIEW_DEFINITION from INFORMATION_SCHEMA.VIEWS where TABLE_NAME = '%s';", viewName)
+		if err = d.db.QueryRow(query).Scan(&definition); err != nil {
+			return nil, err
+		}
+		ddls = append(ddls, fmt.Sprintf("CREATE OR REPLACE VIEW %s AS %s", viewName, definition))
```

Comment:
I assume this means users need to specify `CREATE OR REPLACE VIEW` in a schema definition to satisfy `strings.ToLower(currentView.definition) == strings.ToLower(desiredView.definition)`.

But I think `OR REPLACE` sounds like an implementation detail to me, and because other resources like tables are (at least documented to be) declared as `CREATE TABLE`, I think VIEW should be declared with `CREATE VIEW` without OR REPLACE for consistency across sqldef.

Or do you have any motivation to prefer `CREATE OR REPLACE VIEW` in a schema file rather than `CREATE VIEW`? Ideally sqldef should support both `CREATE VIEW` and `CREATE OR REPLACE VIEW`, but if we were to choose one, I'd like to choose `CREATE VIEW`.

Reply:
Thank you for your reviewing! i fixed it!
https://github.com/k0kubun/sqldef/pull/78/commits/c5287ad24736e8df22414dc96b13202141c58688

Reply:
Oops, I was misunderstanding your implementation. I thought `currentView.definition` includes the `CREATE OR REPLACE VIEW`, but it wasn't. So this part didn't really matter. Anyway the change makes sense for similar reasons, thanks!

Reply:
Since you added the commit, let me be consistent everywhere https://github.com/k0kubun/sqldef/pull/78/commits/ec553382cbbdf49dbe06d916ef7d7dd70b192350.